tests/test_pyxis.py

```diff
@@ -601,6 +601,36 @@ def test_is_bundle_false(self, page):
         is_bundle = self.px.is_bundle("foobar-bundle-1-234")
         self.assertFalse(is_bundle)
 
+    @patch('freshmaker.pyxis.Pyxis._pagination')
+    def test_is_hotfix_image_true(self, page):
+        page.return_value = [{
+            "parsed_data": {
+                "labels": [
+                    {"name": "com.redhat.hotfix", "value": True},
```

Comment:
`value` should be some string, please use a string instead, not sure about the real format, you can just give it something like `1.2hotfix`.

Reply:
Found an image with an actual hotfix label - https://brewweb.engineering.redhat.com/brew/archiveinfo?archiveID=5886789
I'll use it for the test value